NeuroID/src/androidLib/java/com/neuroid/tracker/events/NIDTouchEventManager.kt

```diff
@@ -214,23 +227,23 @@ class NIDTouchEventManager(
                     }
                 }
 
-               /* if (type.isNotEmpty()) {
-                    getDataStoreInstance()
-                        .saveEvent(
-                            NIDEventModel(
-                                type = type,
-                                tg = hashMapOf(
-                                    "etn" to currentView?.javaClass?.simpleName.orEmpty(),
-                                    "tgs" to nameView,
-                                    "sender" to currentView?.javaClass?.simpleName.orEmpty()
-                                ),
-                                tgs = nameView,
-                                ts = timeMills,
-                                gyro = gyroData,
-                                accel = accelData
-                            )
-                        )
-                }*/
+                /* if (type.isNotEmpty()) {
```

Comment:
consider removing.

Reply:
there are a lot of these peppered through the code. too many to list. consider going through quick and removing commented code.